dev-tools.sh

```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=960eba39ed9543b17654dbd5e06f3a2ab2b1fb4b
```

Comment:
Revert it back after dev_tools release and before merging this PR.

Reply:
Changes reverted